File: data/conala-train_code-davinci-002_1666524963/Python/368.py
def sum_of_product_of_combinations(lst):
    """
    Given a list of numbers, return the sum of the product of all possible combinations of two numbers.
    """
    return sum([x*y for x in lst for y in lst if x != y])

print(sum_of_product_of_combinations([1,2,3,4,5]))

File: data/conala-train_code-davinci-002_1666524963/Java/368.java
public static int sumOfProductOfCombinations(List<Integer> list) {
    int sum = 0;
    for (int i = 0; i < list.size(); i++) {
        for (int j = i + 1; j < list.size(); j++) {
            sum += list.get(i) * list.get(j);
        }
    }
    return sum;
}